### PR TITLE
Prepare documentation for release of concordium-client 6.3.0

### PR DESCRIPTION
**Pull request by DOBEN**
## Purpose

DO NOT MERGE

## Changes

DO NOT MERGE

### CODE REVIEW

File: source/mainnet/net/installation/downloads.rst [hunk 1]
```diff
@@ -167,13 +167,13 @@ Concordium Client |client-version|
...
 
--  `Download the Mainnet Concordium Client for macOS <https://distribution.concordium.software/tools/macos/signed/concordium-client-6.2.1-0.pkg>`_
+-  `Download the Mainnet Concordium Client for macOS <https://distribution.concordium.software/tools/macos/signed/concordium-client-6.3.0-1.pkg>`_
 
    - The macOS distribution is an installer that places an alias to the binary
      into the folder ``/usr/local/bin``. So after installing, you should have
      ``concordium-client`` on your path.
 
--  `Download the Mainnet Concordium Client for Windows <https://distribution.concordium.software/tools/windows/signed/concordium-client_6.2.1-0.exe>`_
+-  `Download the Mainnet Concordium Client for Windows <https://distribution.concordium.software/tools/windows/signed/concordium-client_6.3.0-1.exe>`_
```
Comment (td202) [inline source/mainnet/net/installation/downloads.rst:176]:
```suggestion
-  `Download the Mainnet Concordium Client for Windows <https://distribution.concordium.software/tools/windows/signed/concordium-client_6.3.0-1.zip>`_
```

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 2]
```diff
@@ -1643,37 +1643,61 @@ Tools
...
+
+    - Add command ``transaction add-signature`` to add a signature to a partially-signed transaction.
+
+    - Add optional ``--out`` flag to all transaction-creating commands to output a partially-singed transaction to a file.
+
+    - Add preliminary support for protocol version 7.
+
+    - Update GHC version to 9.6.4 (lts-22.9).
+
+    - Update Rust version to 1.73.
```
Comment (td202) [inline source/mainnet/net/release-notes/release-notes-lp.rst:1666]: I think we can remove these, as they're not really user visible.

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 3]
```diff
@@ -1643,37 +1643,61 @@ Tools
...
 
-        - Added additional configuration options that use validator in place of baker in accordance with the upcoming tokenomics changes. For example, ``concordium-client validator add``. The older options still exist, but are hidden.
+    - Remove command ``transaction send-shielded`` to disable the transfer of CCD from the shielded balance of the account to the shielded balance of another account.
 
-        - The ``module inspect`` command now attempts to print any embedded verifiable build information.
+    - Remove command ``account shield`` to disable the transfer of CCD from the public balance to the shielded balance of an account.
+
+    - Add command ``transaction add-signature`` to add a signature to a partially-signed transaction.
+
+    - Add optional ``--out`` flag to all transaction-creating commands to output a partially-singed transaction to a file.
```
Comment (td202) [inline source/mainnet/net/release-notes/release-notes-lp.rst:1660]:
```suggestion
    - Add optional ``--out`` flag to all transaction-creating commands to output a partially-signed transaction to a file.
```

**Review by td202 — APPROVED** [review]

**Review by mh-concordium — APPROVED** [review]
I have checked the links, all good.

**Pull request merged**